### PR TITLE
Request developer to include gif or screenshot showing changes they made

### PR DESCRIPTION
**Pull request by kristinochka**
Signed-off-by: KRISTINA MAYO <ktopchi@us.ibm.com>

## Proposed changes

There has been instances in the past where is was not clear how the code in a PR would change the User Experience even after lengthy description was included. I propose to add a guideline so a user who makes UI changes would need to include either a gif (preferably) or a screenshot (if a change is really simple). That way, the reviewer can quickly see what changes has been made without even loading the code locally and running it. 

## Release Notes
Request developer to include gif or screenshot showing changes they made

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] `npm run vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kristinochka** [comment]
@lauren-li @zFernand0 Please review. I'm not able to add reviewers in the **Reviewers** section on the PR, therefore, I mentioned you here.

**Comment by jellypuno** [comment]
Maybe adding this to the contributing guidelines as well would be a good idea? What do you think?
https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance

**Comment by kristinochka** [comment]
I think it would not hurt :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ _Put an `x` in the boxes that apply. You can also fill these out after creating
 - [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
 - [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
 - [ ] PR Description is included
+- [ ] gif or screenshot is included if visual changes are made
```
Comment (lauren-li) [inline .github/PULL_REQUEST_TEMPLATE.md:30]: I think this is a good idea! It would definitely make some PRs easier to review and help expedite the review process for those PRs.

Just for clarity and to make the capitalization of the list items more consistent, how about rewording to:
> Screenshot or gif is included in PR description if visual changes are made

@jellypuno I agree with posting this in the contributor guidelines, as well. Who would be responsible for doing that?

Comment (zFernand0) [reply .github/PULL_REQUEST_TEMPLATE.md:30]: I was going to say that it could've been done in this same PR by updating the [Contributin.md](https://github.com/zowe/vscode-extension-for-zowe/blob/master/CONTRIBUTING.md) file, but we've never updated that file (personally I think we should since that's the default recommendation by GitHub when people open PRs)
We can tackle that as a documentation GH issue, and update the [wiki information](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) with that.

**Comment by zFernand0** [comment]
Hey @kristinochka, could you request to be included in the Zowe Org?
I remember you were part of Zowe a while back 😋 
https://github.com/orgs/zowe/teams/vscode-team/members

@dkelosky, `build` (or what should be `theia-build` isn't running on forks 🤔 
thoughts?

**Comment by zFernand0** [comment]
PR looks good and there is an issue (#950) to address the wiki and or contributing updates.

**Comment by zFernand0** [comment]
> I like this template change. It could also prompt us to check if the gifs in the readme and doc need to be updated.

noted in #950

**Comment by kristinochka** [comment]
> Hey @kristinochka, could you request to be included in the Zowe Org?
> I remember you were part of Zowe a while back 😋
> https://github.com/orgs/zowe/teams/vscode-team/members

How do I request to be included in the org? The url provided returns 404 for me.

**Comment by zFernand0** [comment]
Maybe @jackjia-ibm or @MarkAckert can help here?
